model/src/lib.rs

```diff
@@ -529,6 +530,55 @@ impl str::FromStr for Identity {
 
 impl_sqlx_type_display_from_str!(Identity);
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Libp2pIdentity(PeerId);
```

Comment:
You could move it into a module `libp2p` and call it `PeerId`.

Reply:
I did that which caused a bit of refactoring. We use `peer_id` throughout the code base now, except when deriving the identity using the prefix `LIBP2P_IDENTITY`. I can't change that because it would change our identity.